CHANGES.md

```diff
@@ -40,6 +40,7 @@
 - Trim the data of the `url` file before filling the `url.src` field. This fixes an issue that caused the `url.src` field to be a multi-line string instead of single line. (#270, @gpetiot)
 - Fix a bug causing dune-release to exclude all hidden files and folders (starting with `.`) at the
   repository from the distrib archive (#298, @NathanReb)
+- Do not assume the main package is the one with the shortest name (#267, @gpetiot)
```

Comment:
Is that what we were doing before? I thought what we were doing before was in some places (`prepare`) to take the package with the shortest name, but in other places (`submit`) even just the first package in the list (leading to the problem described in #265 of taking the synopsis of `tls-mirage` instead of the one from `tls` for the PR message). I think what you are doing now is to infer the main package from the dune-project; if that fails, take the one with the shortest name, if that's unique, and if it's not unique, infer it from the README, no? Maybe you can put the sentence without negation, something like "Infer the main package from the dune-project, if possible".

Reply:
That's a better changelog entry indeed, that should be less confusing to users